docs/entity_alarm_control_panel.md

```diff
@@ -11,15 +11,39 @@ sidebar_label: Alarm Control Panel
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| code_format | string | `None` | The format of the panel code.
+| state	| string | **Required** | One of the states listed in the states section.
+| code_format | string | `None` | Regex for code format or `None` if no code is required.
+| changed_by | string | `None` | Last change triggered by.
 
-### Available code formats
 
-| Value | Description
+## States
+| State | Description
 | ----- | -----------
-| None | No code required.
-| Number | Code is a number (Shows ten-key pad on frontend).
-| Any | Code is a string.
+| `SERVICE_ALARM_TRIGGER` | The alarm is triggered.
```

Comment:
This is wrong. It needs to be the actual states and not the constants. Also these are the service constants?

Reply:
I just followed the implementation of manual.py (but must have confused the STATE and SERVICE constraints). 

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/alarm_control_panel/manual.py

Reply:
Yeah, so it needs to be the actual values of `STATE_ALARM_DISARMED`, which is `disarmed`